frontend/src/features/admin/edit-user/EditUserForm.tsx

```diff
@@ -101,7 +101,7 @@ const EditUserForm: React.FunctionComponent<IEditUserFormProps> = ({
             </TooltipWrapper>
           </SectionField>
 
-          <SectionField label="MoTI Region(s)" required wideScreen>
+          <SectionField label="MoTI Region(s)" required labelWidth="2">
```

Comment:
this is fine, but please make sure to leave a note for Sue to make sure she runs regression on these screens this sprint.